functions.php

```diff
@@ -21,17 +21,90 @@ function getCarsDb(): array
 
 function displayCars(array $getCars): string
 {
-    $display =  '<div class="section">';
-    foreach ($getCars as $cars) {
-        $display .= '<div class="subSection">'
-            .'<p>Make: <span>'. $cars['make'] .'</span></p>
-                          <p>Model: <span>'. $cars['model'] .'</span></p>
-                          <p>Fuel Type: <span>'. $cars['fuel'] .'</span></p>
-                          <p>Gearbox: <span>'. $cars['gearbox'] .'</span></p>
-                          <p>Year: <span>'. $cars['year'] .'</span></p></div>';
+    $display = '<div class="section">';
+    if (empty($getCars)) {
+        $display .= '<p>Something went wrong!</p>';
+    } else {
+        foreach ($getCars as $cars) {
+            $display .= '<div class="subSection">' .
+                '<p>Make: <span>' . $cars['make'] . '</span></p>' .
+                '<p>Model: <span>' . $cars['model'] . '</span></p>' .
+                '<p>Fuel Type: <span>' . $cars['fuel'] . '</span></p>' .
+                '<p>Gearbox: <span>' . $cars['gearbox'] . '</span></p>' .
+                '<p>Year: <span>' . $cars['year'] . '</span></p>' .
+                '</div>';
+        }
     }
     $display .= '</div>';
-
     return $display;
 }
 
+function validate($postData, $db)
+{
+    $errors = ['make' => '',
+        'model' => '',
+        'fuel' => '',
+        'gearbox' => '',
+        'year' => ''
+    ];
+
+    if (empty($postData['make'])) {
+        $errors['make'] = 'Please add a make!';
+    }
+
+    if (empty($postData['model'])) {
+        $errors['model'] = 'Please add a model name!';
+    } else {
+        $query = $db->prepare('SELECT `model` FROM `cars` WHERE `model` = ?');
+
```

Comment:
these new lines are unnecessary. Because line 58, 60 and 62 are all part of one database query it's probably better to have them grouped together for clarity. Please remove the lines in between.